store/sqlstore/post_store.go

```diff
@@ -1366,3 +1367,68 @@ func (s *SqlPostStore) GetRepliesForExport(parentId string) store.StoreChannel {
 		}
 	})
 }
+
+func (s *SqlPostStore) GetDirectPostParentsForExportAfter(limit int, afterId string) store.StoreChannel {
+	return store.Do(func(result *store.StoreResult) {
+		query := getQueryBuilder(s).
+			Select("p.*", "Users.Username as User").
+			From("Posts p").
+			Join("Channels ON p.ChannelId = Channels.Id").
+			Join("Users ON p.UserId = Users.Id").
+			Join("ChannelMembers cm ON cm.ChannelId = Channels.Id").
+			Where(sq.And{sq.Gt{"p.Id": afterId},
+				sq.Eq{"p.ParentId": string("")},
+				sq.Eq{"p.DeleteAt": int(0)},
+				sq.Eq{"Channels.DeleteAt": int(0)},
+				sq.Eq{"Users.DeleteAt": int(0)},
+				sq.Eq{"Channels.Type": []string{"D", "G"}}}).
+			GroupBy("Channels.Id, p.Id, Users.Username").
```

Comment:
This `GroupBy` is redundant once we eliminate the spurious `ChannelMembers` `JOIN` above.

Reply:
Indeed. Fixed.